launch_ros/launch_ros/actions/set_ros_log_dir.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the `SetROSLogDir` action."""
+
+import os
+from typing import List
+
+from rclpy.logging import get_logging_directory
+
+from launch import Action
+from launch import Substitution
+from launch.frontend import Entity
+from launch.frontend import expose_action
+from launch.frontend import Parser
+from launch.launch_context import LaunchContext
+from launch.some_substitutions_type import SomeSubstitutionsType
+from launch.utilities import normalize_to_list_of_substitutions
+from launch.utilities import perform_substitutions
+
+
+@expose_action('set_ros_log_dir')
+class SetROSLogDir(Action):
```

Comment:
Consider renaming for consistency with other actions (e.g. `PushRosNamespace`):

```suggestion
class SetRosLogDir(Action):
```

Reply:
Eww, that's gross, but I will change it for consistency I guess. `Ros` is not `ROS`... I know that's the "Google" way in C++, but honestly it's dumb to me to only capitalize the first letter in an acronym.

Reply:
I agree. Also [PEP 8](https://peps.python.org/pep-0008/#naming-conventions) agrees.

We could consider keeping it and adding aliases for the other class names.

Reply:
Hmm, ok let's do that.

Reply:
I changed it back. I'll open another pr to change the class names, and yet another pr to deprecate the old names.